drivers/can/can_net.c

```diff
@@ -183,6 +186,33 @@ static inline int attach_mcast_filter(struct net_can_context *ctx)
 	return filter_id;
 }
 
+#ifdef CONFIG_NET_L2_CANBUS_ETH_TRANSLATOR
+static inline int attach_eth_bridge_filter(struct net_can_context *ctx)
+{
+	static struct zcan_filter filter = {
+		.id_type = CAN_EXTENDED_IDENTIFIER,
+		.rtr = CAN_DATAFRAME,
+		.rtr_mask = 1,
+		.ext_id_mask = CAN_NET_IF_ADDR_DEST_MASK,
+		.ext_id = (NET_CAN_ETH_TRANSLATOR_ADDR <<
+			   CAN_NET_IF_ADDR_DEST_POS)
+	};
+
+	int filter_id;
+
+	filter_id = can_attach_isr(ctx->can_dev, net_can_recv,
+				   ctx, &filter);
+	if (filter_id == CAN_NET_FILTER_NOT_SET) {
+		NET_ERR("Can't attach ETH bridge filter.");
+		return CAN_NET_FILTER_NOT_SET;
+	}
+
+	NET_DBG("Attached ETH bridge filter %d.", filter_id);
+
+	return filter_id;
+}
+#endif /*CONFIG_NET_L2_CANBUS_ETH_TRANSLATOR*/
```

Comment:
Another minor nitpick. Instead of `/*xxx*/`, could you write these comments with spaces after and before comment, so `/* xxx */`